### ISSUE TITLE
strings.xml not updated to 5.6.0

### ISSUE THREAD

**Issue by shaubert** [onfido/onfido-android-sdk#110]
In migration doc you wrote that two new strings were added:
onfido_mrz_not_detected_title
onfido_mrz_not_detected_subtitle

but both of them are missing in https://github.com/onfido/onfido-android-sdk/blob/master/strings.xml

**Comment by shaubert** [comment]
Also onfido_accessibility_camera_face_capture_view is missing as well as almost all onfido_accessibility_** strings

**Comment by burakeregar** [comment]
Hi @shaubert, thanks for the report!

It should be fixed now.
